### PR TITLE
Readme

### PR DESCRIPTION
**Pull request by iqis**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,12 @@
 # rolldown
 
 [![Travis build status](https://travis-ci.com/yihui/rolldown.svg?branch=master)](https://travis-ci.com/yihui/rolldown)
+![](https://img.shields.io/badge/lifecycle-experimental-orange.svg)
```
Comment (yihui) [inline README.md]: Please try your best to add the alt text to images whenever it is possible (consider visually impaired people): https://yihui.name/cn/2017/04/img-alt/

File: README.md [hunk 2]
```diff
@@ -1,3 +1,12 @@
...
+
+An extension to [R Markdown](https://rmarkdown.rstudio.com/) that builds scroll-driven HTML documents for storytelling, a.k.a _scrollytelling_, powered by JavaScript libraries. Try here:
+```r
```
Comment (yihui) [inline README.md]: Do you remember that I bowed to one presenter with apologies for nitpicking the insufficient spaces and empty lines at the unconf?

Please add an empty line before this line. Reasons: https://yihui.name/cn/2017/05/blank-line/

File: README.md [hunk 3]
```diff
@@ -1,3 +1,12 @@
...
+## Install from GitHub
+remotes::install_github("uncoast-unconf/rolldown")
+```
+
+To apply Scrollama to the document, use `scrollama` or `scrollama_sidebar` in the `output` field of the document heading; Include `scrollama_setup()` in an R code block at the end of the document to adjust specifications.
```
Comment (yihui) [inline README.md]: For `scrollama_sidebar()`, it is not necessary to use `scrollama_setup()`.

Here I'd rather include a brief working example for each format.

File: README.md (hunk 1 above)
Comment (iqis) [reply README.md]: I will take this by heart.

File: README.md (hunk 2 above)
Comment (iqis) [reply README.md]: I actually overlooked this when I threw together the draft. 
Normally when I use Markdown, I like to put two empty lines prior to a level 1 header, and one empty line for the rest of the headers and other elements such as code blocks. I hope this is OK.

File: README.md (hunk 3 above)
Comment (iqis) [reply README.md]: Noted and done.

File: README.md [hunk 4]
```diff
@@ -1,12 +1,71 @@
...
+
+It is easy to adopt Scrollama for a document. A minimal example goes as follows: 
+
+
+````
+---
+title: "Scrollama Document"
+output:
+  rolldown::scrollama_sidebar:
+  self_contained: no
```
Comment (yihui) [inline README.md]: Please indent this line (or remove it, since it is not required).

File: README.md [hunk 5]
```diff
@@ -1,12 +1,71 @@
...
+---
+```{r echo = FALSE}
```
Comment (yihui) [inline README.md]: Empty line before this line please.

File: README.md [hunk 6]
```diff
@@ -1,12 +1,71 @@
...
+# include this code block in the beginning to specify a style
+rolldown::rolldown_style_default()
+```
+
+# Level-One
+Use level-one heading to designate scrolling sections.
```
Comment (yihui) [inline README.md]: And before this line.

File: README.md [hunk 7]
```diff
@@ -1,12 +1,71 @@
...
+```{r echo = FALSE}
+# include this code block in the beginning to specify a style
+rolldown::rolldown_style_default()
+```
+
+# Level-One
+Use level-one heading to designate scrolling sections. 
+
+## Level-Two
+Level-two and below headings...
```
Comment (yihui) [inline README.md]: And this.

File: README.md [hunk 8]
```diff
@@ -1,12 +1,71 @@
...
+
+### Level-Three
+...are all contained in the same section.
```
Comment (yihui) [inline README.md]: And this... I'm not going to repeat it in the rest of the example.

File: README.md [hunk 9]
```diff
@@ -1,12 +1,71 @@
...
+
+# Image
+Images are treated the same with plots. Use Markdown syntax to insert them in texts.   
+
+Local news [reported](https://www.desmoinesregister.com/story/news/2018/04/27/rubber-duck-des-moines-yess-duck-derby/558852002/) a giant inflatable rubber duck rolling down the street of Des Moines, where this package is conceived.  
+![The Rolling Duck](https://www.gannett-cdn.com/-mm-/29c397077fc2506ed005328c6d03b4cc695265f2/c=3-0-483-361/local/-/media/2018/04/27/IAGroup/DesMoines/636604380015350378-duck.JPG?width=534&height=401&fit=crop)  
+
+
+# HTML Widgets
+Support for HTML Widgets are under development.
```
Comment (yihui) [inline README.md]: `are` -> `is`

File: README.md (hunk 2 above)
Comment (yihui) [reply README.md]: Sure.

File: NAMESPACE [hunk 10]
```diff
@@ -1,5 +1,6 @@
 # Generated by roxygen2: do not edit by hand
 
+export(rolldown_style_default)
```
Comment (yihui) [inline NAMESPACE]: Could you do only one thing per PR and not commit this change? https://yihui.name/en/2018/02/bite-sized-pull-requests/

**Comment by iqis** [comment]
I have tried, but was still a little wary of reverting.  Since this is branch is isolated, I thought I might just resetting it to a prior commit, start afresh and force push.

**Pull request merged**